arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json

```diff
@@ -238,7 +254,7 @@
           "CognitiveServicesAccounts"
         ],
         "description": "Returns all the resources of a particular type belonging to a resource group",
-        "operationId": "CognitiveServicesAccounts_ListByResourceGroup",
+        "operationId": "Accounts_ListByResourceGroup",
```

Comment:
@anudeepsharma , alright then in this case we will initiate a new version of swagger. I will do it in another PR, please abandon this one.